modules/server/src/main/ts/bedrock/cli/Extraction.ts

```diff
@@ -35,6 +35,19 @@ export const any = <E, A> (name: string, value: A): Attempt<E, A> => {
   return Attempt.passed(value);
 };
 
+export const nat = (name: string, value: string): Attempt<string[], number> => {
+  const failed = Attempt.failed<string[], number>(['Invalid value for property: ' + name + ': [' + value + '] is not a natural number']);
```

Comment:
Could this use a template string instead?

Reply:
Also wondering if instead of saying `natural number` if it'd be clearer to say something like `must be a number greater than 0`?

Reply:
I got my numbers mixed up. This is a positive integer, not a nat.

Reply:
Fixed.

Reply:
Template strings fixed.